src/main.js

```diff
@@ -218,6 +218,13 @@ allowProtocolDialog.init(mainWindow);
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 app.on('ready', function() {
+  ipcMain.on('notified', function(event, arg) {
```

Comment:
I don't think we can have ```ipcMain.on('notified', function(event, arg)``` two times. If this even works, it's likely that one overrides the other, which isn't really what we want here. As that would mean, the window will not flash, when you set shouldShowTrayIcon to true.